src/algebra/big_operators/basic.lean

```diff
@@ -1013,6 +1014,11 @@ calc ∏ a in s, f (g a)
 ... = ∏ b in s.image g, f b ^ (s.filter (λ a, g a = b)).card :
   prod_congr rfl (λ _ _, prod_const _)
 
+@[to_additive]
+lemma prod_comp_of_injective [decidable_eq γ] (f : γ → β) (g : α → γ) (hg : function.injective g) :
+  s.prod (f ∘ g) = (s.image g).prod f :=
+by { erw [← s.prod_map ⟨g, hg⟩ f, map_eq_image], congr, }
```

Comment:
Isn't this a trivial consequence of `finset.prod_image`?

Reply:
Oh, right!

Thanks.